devops/iam-did-auth-proxy-dev/values.yaml

```
@@ -74,14 +74,14 @@ did-auth-proxy-helm:
 
   opsValues:
     RPC_URL: https://volta-rpc.energyweb.org/
-    CACHE_SERVER_URL: https://identitycache-dev.energyweb.org/v1
+    CACHE_SERVER_URL: https://identitycache-gp4btc-dev.energyweb.org/v1
```

There is no https://identitycache-gp4btc-dev.energyweb.org/v1 there is only https://identitycache-dev.energyweb.org/v1
```suggestion
    CACHE_SERVER_URL: https://identitycache-dev.energyweb.org/v1
```